<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>351</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="DB_button">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Прикрепить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="File_button">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Прикрепить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QLabel" name="DB_file">
       <property name="text">
        <string>Файл БД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="File_file">
       <property name="text">
        <string>Файл Attach</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="DB_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="File_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="email_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать отправку</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="DB_file_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Прогресс отправки</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_fds">
    <property name="title">
     <string>TG: @Tachg</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuE_mail_nikolay_krutykh_yandex_ru">
    <property name="title">
     <string>E-mail: nikolay.krutykh@yandex.ru</string>
    </property>
   </widget>
   <addaction name="menu_fds"/>
   <addaction name="menuE_mail_nikolay_krutykh_yandex_ru"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
